build/contracts/Proxy.json

```diff
@@ -0,0 +1,1006 @@
+{
```

Comment:
Is this needed? It can be imported like this:
`import SafeProxy from '@gnosis.pm/safe-contracts/build/contracts/Proxy.json'`